website/content/api-docs/acl/binding-rules.mdx

```diff
@@ -200,7 +204,10 @@ queries](/nomad/api-docs#blocking-queries) and [required ACLs](/nomad/api-docs#a
 
 - `Selector` `(string: "")` - A boolean expression that matches against verified
   identity attributes returned from the auth method during login.  This is
-  optional and when not set, provides a catch-all rule.
+  optional and when not set, provides a catch-all rule. If set, it must be a
+  valid [go-bexpr][] expression; for example, a dash in the claim name will
+  require it to be encased in quotes and escaped such as
+  `"\"project-developer\" in list.roles"`.
```

Comment:
For my own clarity, the escaping is for the shell or JSON body, right? If I was using the command line can I do `-selector '"product-developer" in list.roles'` and that could get escaped when it's serialized to JSON?

Reply:
Yes exactly. I use the double quote escape example as it seems more obvious and visible what is going on to me.

```
$ nomad acl binding-rule create \
    -description "example binding rule" \
    -auth-method "auth0" \
    -bind-type "role" \
    -bind-name "eng-ro" \
    -selector '"product-developer" in list.roles'
ID           = 5a6dba47-0c6c-5d3c-98de-cbb06ed97c23
Description  = example binding rule
Auth Method  = auth0
Selector     = "\"product-developer\" in list.roles"
Bind Type    = role
Bind Name    = eng-ro
Create Time  = 2023-02-17 13:51:47.929953 +0000 UTC
Modify Time  = 2023-02-17 13:51:47.929953 +0000 UTC
Create Index = 17
Modify Index = 17
```